modules/meta/standard.nix

```diff
@@ -30,11 +31,11 @@ in
 
   config = lib.mkIf cfg.enable {
     environment.systemPackages = cfg.packages;
-    services.pyhesiodfs.enable = true;
-    security.krb5.enable = true;
+    services.pyhesiodfs.enable = lib.mkDefault true;
     services.openafsClient = {
```

Comment:
AFS config should probably be in its own file with its own enable option, just like you've done for Kerberos.